packages/eslint-config-generator/src/config-generator.ts

```diff
@@ -83,7 +83,7 @@ function getBaseConfig(options: Options): ESLintConfig {
             '**/postcss.config.{js,ts}', // postcss config
             '**/tailwind.config.{js,ts}', // tailwind config
             '**/vite.config.{js,ts}', // vite config
-            '**/prettier.config.{js,ts}', // prettier config
+            '**/*prettier{.config, rc}.{js,ts,cjs}', // prettier config
```

Comment:
```suggestion
            '**/prettier.config.{js,cjs,ts}', // prettier config
            '**/.prettierrc.{js,cjs,ts}', // prettier config
```

Easier to read and maintain IMO

Reply:
Done @Kocal ;)